### PR TITLE
Adds team objectives to the cult antag panel

### PR DESCRIPTION
**Pull request by SuperNovaa41**
## About The Pull Request

Cult wasn't showing the objectives for new joins, this displays the team's objectives so that admins can see what objectives the cultist might actually have, instead of digging through VV to find them.

## Why It's Good For The Game

Admins deserve SOME things i guess

## Changelog

:cl:
admin: The antag panel will display the cult's team objectives
/:cl:

### CODE REVIEW

File: code/modules/antagonists/cult/cult.dm [hunk 1]
```diff
@@ -202,6 +202,17 @@
...
 			qdel(o)
 
+/datum/antagonist/cult/antag_panel_objectives()
+	. = ..()
+
+	. += "<br>"
+	. += "<i><b>Team Objectives:</b></i><br>"
+	var/obj_count = 1
+	for (var/datum/objective/objective as anything in cult_team.objectives)
+		. += "<B>[obj_count]</B>: [objective.explanation_text] <br>"
```
Comment (dragomagol) [inline code/modules/antagonists/cult/cult.dm:212]:
```suggestion
		. += "<B>[obj_count]</B>: [objective.explanation_text]<br>"
```

**Comment by dragomagol** [comment]
How does this look in-game?

**Comment by SuperNovaa41** [comment]
image

in this case which the user only gets team objectives, these won't be duplicated

**Comment by JohnFulpWillard** [comment]
image

Team objectives are already shown?

**Comment by MrStonedOne** [comment]
> image
> 
> Team objectives are already shown?

image
not always.

Which is why both have to be shown.

**Comment by MrStonedOne** [comment]
> Alt to https://github.com/tgstation/tgstation/pull/69746

Both prs are necessary. We need to be specific about what we are displaying to admins when its information we use to decide when to ban or not ban people.

if we are storing objectives information in two seperate places (team vs individual) with edge cases where team objectives are sometimes, but not always being duplicated to individual objectives, then we do need to specify to the admins what they are seeing in the interfaces.

**Comment by SuperNovaa41** [comment]
> > Alt to #69746
> 
> Both prs are necessary. We need to be specific about what we are displaying to admins when its information we use to decide when to ban or not ban people.
> 
> if we are storing objectives information in two seperate places (team vs individual) with edge cases where team objectives are sometimes, but not always being duplicated to individual objectives, then we do need to specify to the admins what they are seeing in the interfaces.

okay thats fine

**Comment by SuperNovaa41** [comment]
Closing this because it hides the greater issue which is, cult objectives are always added to personal objectives.
image
image

If they're not showing up that issue should be tacked directly.

**Pull request closed without merging**